week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -37,7 +62,21 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(arr) {}
+function formatPercentage(arr) {
+  let finalArr = [];
+  for ( i in arr ){
+    let x = arr[i];
+    if (x > 100){
+      x = 100
+    }
+    if ( x % 1 !== 0 ){
+    x = (Math.round(x * 100) / 100).toFixed(2)
+    }
```

Comment:
Wondering what's the additional value of the if statement?

Reply:
I didn't understand the question!

Reply:
I'll take a 2nd look at, might have missed something.

Reply:
Let me ask another way. When will the expression in the IF statement be FALSE?
Can you say me an example when it wouldn't be true?

Reply:
this condition will evaluate to false when a decimal number is passed to it, as the reminder/modulos operator will always return the fractional part of the number if it's devided by 1.
for example if we do (10.5 % 1 ) this will return 0.5 which will evaluate to FALSE in our case here.

Reply:
You're absolutely right. I've missed that it can even be a floating number and not only whole number.

Reply:
Tip: `toFixed` function does not only format the number but also rounds it.
It can make your rounding much simpler.

Here is an example: https://www.w3schools.com/jsref/jsref_tofixed.asp
For more: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed